locales/pt_BR/functions.po

```diff
@@ -0,0 +1,494 @@
+#  generated by goflow docgen
+#  
+#, fuzzy
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2020-06-02 17:13-0500\n"
+"Language: pt_BR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+
+msgid "A single random integer in the given inclusive range."
+msgstr ""
+
+msgid "An error is returned if the value can't be converted."
+msgstr ""
+
+msgid "An optional third argument `humanize` can be false to disable the use of thousand separators."
+msgstr ""
+
+msgid "An optional third parameter `group` determines which matching group will be returned."
+msgstr ""
+
+msgid "Calculates the date value arrived at by adding `offset` number of `unit` to the `datetime`"
+msgstr ""
+
+msgid "Capitalizes each word in `text`."
+msgstr ""
+
+msgid "Converts `date` to a UNIX epoch time."
+msgstr ""
+
+msgid "Converts `text` into something that can be read by IVR systems."
+msgstr ""
+
+msgid "Converts `text` to lowercase."
+msgstr "Converte `text` em minúsculas."
+
+msgid "Converts `text` to uppercase."
+msgstr "Converte `text` em maiúsculas."
```

Comment:
The `text` part is a reference to an arg in the display function signature. We almost certainly don't want to go the Excel route of translating function names, but you could conceivably translate args in the signature.. but then maybe that's more confusing because the function name is in English but the args are translated, e.g. `upper(texto)`. So for now I'm assuming these stay untranslated.